### ISSUE TITLE
Bug: Argument of type 'ICanvas' is not assignable to parameter of type 'Node'.

### ISSUE THREAD

**Issue by ooyay** [pixijs/pixijs#9253]
### Current Behavior

using typescript, trying to append `app.view` to body but am getting type error.

```ts
const app = new PIXI.Application({ width: 640, height: 360 })
document.body.appendChild(app.view)
```

<img width="718" alt="image" src="https://user-images.githubusercontent.com/91382501/224171400-5f7db016-e120-41fb-a109-2f03373c3e4a.png">

### Expected Behavior

YES its possible to set `app.view as any` but i guess this should be fix.

### Steps to Reproduce

```html
<script>
  import * as PIXI from 'pixi.js'

  const app = new PIXI.Application({ width: 640, height: 360 })
  document.body.appendChild(app.view)
</script>
```

### Environment

- **`pixi.js` version**: *e.g. 7.1.0*
- **Browser & Version**: *e.g. Chrome 108*
- **OS & Version**: *e.g. Ubuntu 22.04*
- **Running Example**: *e.g. <https://pixiplayground.com/>*

### Possible Solution

`as any`

**Comment by bigtimebuddy** [comment]
This was addressed in #8820 (7.0.3+)

```ts
const app = new PIXI.Application<HTMLCanvasElement>({ width: 640, height: 360 })
document.body.appendChild(app.view);
```

**Comment by ooyay** [comment]
> This was addressed in #8820 (7.0.3+)
> 
> ```ts
> const app = new PIXI.Application<HTMLCanvasElement>({ width: 640, height: 360 })
> document.body.appendChild(app.view);
> ```

<img width="144" alt="image" src="https://user-images.githubusercontent.com/91382501/224286466-0a8d1779-9ffc-4c9c-905d-e5d8be764713.png">
am on 7.1.4

**Comment by dreadwail** [comment]
I'm using version `7.2.4`: 

```ts
import { Application } from 'pixi.js';

const application = new Application();

document.body.appendChild(application.view);
```

<img width="1012" alt="Screenshot 2023-06-05 at 10 43 04 AM" src="https://github.com/pixijs/pixijs/assets/712322/cb024946-d7e8-4843-ad0b-2c0db53ab724">

```sh
$ yarn list --pattern @pixi/app
yarn list v1.22.19
└─ @pixi/app@7.2.4
```

However, that being said, this is based on the code examples in the README and on the guides website.  Neither of those places pass a generic to the constructor of `Application`.

If I start passing a generic parameter as seen in the comments thread above, then things work normally:

```ts
import { Application } from 'pixi.js';

// pass generic as seen in the comment thread (not on the website)
const application = new Application<HTMLCanvasElement>();

// no errors now
document.body.appendChild(application.view); 
```

So I think the types are probably correct but I didn't find any code examples guiding me to use generics with this constructor, so the only way that I was able to correct this issue is by finding this github issue comment thread.

At the same time, the guides website seems to be JS only (no TS examples) so I'm not actually sure what the most appropriate venue to communicate this to developers might be. There aren't many TS examples lurking about on the website.

Another possible approach in addition to communicating this somewhere on the website might be to enforce that a generic parameter be passed instead of having it be optional defaulting to `ICanvas`.  This is the current definition:

```ts
export declare class Application<VIEW extends ICanvas = ICanvas> {
```

Since `ICanvas` will never be assignable to `Node` this default parameter does not seem to make much sense to me. 

If it were changed to this then the scenario we're running into would not be possible:

```ts
export declare class Application<VIEW extends ICanvas> {
```

This would result in an error if the developer does not pass a generic.

**Comment by twsau** [comment]
> > This was addressed in #8820 (7.0.3+)
> > ```ts
> > const app = new PIXI.Application<HTMLCanvasElement>({ width: 640, height: 360 })
> > document.body.appendChild(app.view);
> > ```
> 
> Thanks <3

**Comment by christopherball** [comment]
FYI, fellow typescripter here - also got caught on this same `tsc` transpile error until I stumbled upon this thread.  Might be worth mentioning some of these finer points in the official docs, or better yet offering some Typescript examples of the same starter codes...
